### DISCUSSION TITLE
Custom color help

### DISCUSSION THREAD

**Discussion by LAB02-Admin** [Madelena/Metrology-for-Hass#27] [General]
Hey @Madelena,

I've started using your theme and really love it! So clean, and I love how easy you made it to create color variations :) However, I'm a bit stuck at determining exactly what colors to enter. I'd like to add a variation that matches the color I use in Windows (#038387), but I'm not sure what's the best way to get the rgb-primary-color-darker, rgb-primary-color-lighter and accent-color values. 

Are there some tricks or sites that assist in this?

Like I said in my [forumpost](https://community.home-assistant.io/t/hass-agent-windows-client-to-receive-notifications-use-commands-sensors-quick-actions-and-more/369094/162?u=samkr), I suck at graphic design :x

**Comment by Madelena** [comment]
@LAB02-Admin Here's the [Adobe Color wheel method](https://color.adobe.com/nl/create/color-wheel) @legovaer used a while ago, which I thought was pretty clever and consistent: #19 

Basically, you can enter the existing lighter/normal/darker values into the wheel, and then when you move any of the colors, the wheel will automatically calculate the corresponding values of the other two colors for you.

**Comment by LAB02-Admin** [comment]
Neat, thanks! I'll go play with it
